controllers/subscription.go

```diff
@@ -70,34 +69,28 @@ func newSubscription(p api.Pattern) *operatorv1alpha1.Subscription {
 		},
 	}
 
-	if p.Spec.GitOpsConfig.UseCSV {
-		spec.StartingCSV = p.Spec.GitOpsConfig.OperatorCSV
-	}
-
-	if p.Spec.GitOpsConfig.ManualApproval {
-		spec.InstallPlanApproval = operatorv1alpha1.ApprovalManual
-	}
-
-	return &operatorv1alpha1.Subscription{
+	newSubscription = &operatorv1alpha1.Subscription{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "openshift-gitops-operator",
-			Namespace: subscriptionNamespace,
+			Name:      GitOpsDefaultPackageName,
+			Namespace: SubscriptionNamespace,
 		},
-		Spec: spec,
+		Spec: &newSpec,
```

Comment:
Always review your own PR and avoid changes that don't add value

Reply:
Yeah I realize that now.  I used what was in upstream and reverted to previous impl.